app/backend/src/couchers/migrations/versions/d1a68f873ee3_rename_hostrequest_user_columns.py

```diff
@@ -0,0 +1,36 @@
+"""Rename HostRequest user columns
+
+Revision ID: d1a68f873ee3
+Revises: 62fcd41e4dcd
+Create Date: 2021-07-13 01:47:30.120271
+
+"""
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "d1a68f873ee3"
+down_revision = "62fcd41e4dcd"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    ### Manually generated
+    op.alter_column("host_requests", "from_user_id", new_column_name="surfer_id")
+    op.alter_column("host_requests", "to_user_id", new_column_name="host_id")
+    op.alter_column("host_requests", "to_last_seen_message_id", new_column_name="host_last_seen_message_id")
+    op.alter_column("host_requests", "from_last_seen_message_id", new_column_name="surfer_last_seen_message_id")
+
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_constraint("fk_host_requests_from_user_id_users", "host_requests", type_="foreignkey")
+    op.drop_constraint("fk_host_requests_to_user_id_users", "host_requests", type_="foreignkey")
+    op.create_foreign_key(op.f("fk_host_requests_host_id_users"), "host_requests", "users", ["host_id"], ["id"])
+    op.create_foreign_key(op.f("fk_host_requests_surfer_id_users"), "host_requests", "users", ["surfer_id"], ["id"])
+    op.drop_index("ix_host_requests_from_user_id", table_name="host_requests")
+    op.drop_index("ix_host_requests_to_user_id", table_name="host_requests")
+    op.create_index(op.f("ix_host_requests_host_id"), "host_requests", ["host_id"], unique=False)
+    op.create_index(op.f("ix_host_requests_surfer_id"), "host_requests", ["surfer_id"], unique=False)
```

Comment:
Are you not able to just rename the indexes/constraints? This is fine-ish, but creating constraints/indexes can take quite a long time on larger databases and might take a whole table write lock

Reply:
It looks like no and that its an sql thing, not an sqla thing:

https://stackoverflow.com/questions/44631175/alembic-how-to-alter-a-foreigenkey-column 
https://stackoverflow.com/questions/13606469/cannot-change-column-used-in-a-foreign-key-constraint

And yes it requires a table lock (didn't even think of this). Will sqla do that automatically? Or do I need to add it?

Reply:
You can rename them in postgres:

https://www.postgresql.org/docs/current/sql-alterindex.html

https://www.postgresql.org/docs/current/sql-altertable.html

Reply:
I renamed them just now without thinking. It's not about renaming them, it's about applying the same constraint to a newly named column. I don't see a way to do that?

Reply:
What do you mean? The column didn't change, just the name. The constraint still applies to that renamed column

Reply:
Will the index/fk just track the column post-rename?

Reply:
Yep. The names are just labels, the data/indexes/constraints are stored separately